control-plane/stor-port/src/types/v0/store/node.rs

```diff
@@ -154,6 +154,10 @@ pub struct NodeSpec {
     #[serde(skip_serializing_if = "Option::is_none")]
     #[serde(default)] // Ensure backwards compatibility in etcd when upgrading.
     node_nqn: Option<HostNqn>,
+    #[serde(default)] // Ensure backwards compatibility.
+    draining_volumes: Vec<VolumeId>,
+    #[serde(default, skip)] // Ensure backwards compatibility and do not store.
```

Comment:
Don't think you need default here as we have skip?

Reply:
fixed